src/include/btree.i

```diff
@@ -61,6 +66,11 @@ __wt_page_is_empty(WT_PAGE *page)
 static inline bool
 __wt_page_evict_clean(WT_PAGE *page)
 {
+    /*
+     * Be cautious modifying this function: it's reading fields set by checkpoint reconciliation,
```

Comment:
Is it valid to do this? I understand that we won't have a torn write on any of the things that we're checking. But isn't it possible we read `page_state`, then checkpoint writes to `page_state` and `rec_result`, and then we read `rec_result` (i.e. reading a combination that never existed)?